src/cli/dot_command.zig

```diff
@@ -0,0 +1,169 @@
+const std = @import("std");
+const bun = @import("../global.zig");
+const DirInfo = @import("../resolver/dir_info.zig");
+const resolve_path = @import("../resolver/resolve_path.zig");
+const string = bun.string;
+const strings = bun.strings;
+const Output = bun.Output;
+const Global = bun.Global;
+const heap_allocator = bun.default_allocator;
+const system = std.os.system;
+const Run = @import("../bun_js.zig").Run;
+const bundler = @import("../bundler.zig");
+const Command = @import("../cli.zig").Command;
+const RunCommand = @import("./run_command.zig").RunCommand;
+const cli = @import("../cli.zig");
+
+const RunOutput = struct {
+    cancelled: bool,
+    script: ?string,
+};
+
+pub const DotCommand = struct {
+    fn returnFileIfExist(name: string) ?std.fs.File {
```

Comment:
Let's avoid this one-off logic

We can use bun's node_module resolver and behave consistently with regular import/require

https://github.com/oven-sh/bun/blob/5f839c3b5ea93102847ed733e0acd0e378fbaa9c/src/bundler.zig#L384-L403

Reply:
i dont know how to use it

Reply:
I can take a look again but 😕